_topics/en/coinjoin.md

```diff
@@ -21,6 +21,13 @@ primary_sources:
   - title: "Zerolink: the Bitcoin fungibility framework"
     link: https://github.com/nopara73/ZeroLink
 
+  - title: "Privacy Privacy Wiki"
```

Comment:
I think these are secondary sources of aggregated information as opposed to primary, original sources.